NitroxClient/Communication/UdpClient.cs

```diff
@@ -59,6 +60,11 @@ public void ReceivedMessage(object peer)
                             connectedEvent.Set();
                         }
 
+                        if (LargeWorldStreamer.main != null && LargeWorldStreamer.main.IsReady() && !IsConnected)
```

Comment:
Can we do:

`if (NetConnectionStatus.Disconnected)`

This will allow us to have a pop-up during the loading screen and even the color selection screen.